core/decorators/focus-tests-decorator.ts

```diff
@@ -1,8 +1,8 @@
 import "reflect-metadata";
 import { FOCUS } from "./_metadata-keys";
 
-export function FocusTests(constructor: Function) {
+export function FocusTests(constructor: ({ new (...args: Array<any>): any; }) | (() => any)) {
```

Comment:
Also we're probably not returning an any it will always be at least an object I would guess?